app/controllers/api/projects.php

```diff
@@ -133,6 +141,23 @@
             throw new Exception(Exception::PROJECT_RESERVED_PROJECT, "'console' is a reserved project.");
         }
 
+        // TODO: One in 20 projects use shared tables. Temporary until all projects are using shared tables.
```

Comment:
Maybe we can go simpler and after this is ready, just create a single new DB that uses shared tables?